src/sentry/snuba/models.py

```diff
@@ -43,6 +44,11 @@ class Type(Enum):
     type = models.SmallIntegerField()
     dataset = models.TextField()
     query = models.TextField()
+    group_by = ArrayField(
+        models.CharField(max_length=1000),
```

Comment:
Probably the length isn't too important here, but curious why so long? Probably most field names aren't too long

Reply:
https://github.com/getsentry/sentry/blob/0e83f08c78d3b9131dd27a847f0b298e74f525a9/src/sentry/sentry_metrics/consumers/indexer/tags_validator.py#L43

I see two values for max tag length, here `1000` and elsewhere `200`. just going with the upper bounds for now

Reply:
That's for the values right? Not the keys?

Reply:
ah, good point. let me see